openprescribing/frontend/views/bookmark_utils.py

```diff
@@ -859,3 +864,45 @@ def unescape_href(text):
     for href in hrefs:
         text = text.replace(href, html_parser.unescape(href))
     return text
+
+
+def make_all_england_email(bookmark, tag=None):
+    msg = initialise_email(bookmark, 'all-england-alerts')
+    msg.subject = 'Your monthly update on prescribing across NHS England'
+
+    date = ImportLog.objects.latest_in_category('ppu').current_at
+    entity_type = 'CCG'
+    ppu_savings = _cache(_all_england_ppu_savings, entity_type, date)
+    measure_savings = _cache(_all_england_measure_savings, entity_type, date)
+    low_priority_savings = _cache(_all_england_low_priority_savings, entity_type, date)
+    low_priority_total = _cache(_all_england_low_priority_total, entity_type, date)
+    ncso_spending = _first_or_none(
+        ncso_spending_for_entity(None, 'all_england', num_months=1)
+    )
+
+    unsubscribe_path = reverse(
+        'bookmark-login',
+        kwargs={'key': bookmark.user.profile.key}
+    )
+    unsubscribe_link = settings.GRAB_HOST + unsubscribe_path
+
+    context = {
+        'entity_type': entity_type,
```

Comment:
As this is always `"CCG"`, does it need to be passed to the template?

Reply:
I've added a comment to try to explain this:
```
    # This allows us to switch between calculating savings at the practice or
    # CCG level. We use CCG at present for performance reasons but we may want
    # to switch in future.
```